superset-frontend/stylesheets/less/cosmo/bootswatch.less

```diff
@@ -66,9 +66,14 @@
 
 // Buttons ====================================================================
 
+.btn {
```

Comment:
this will change buttons everywhere right? Can we see other screenshots of the ui in addition to SQL Lab?

Reply:
yes let me take a few screenshots